src/app/[locale]/auth/page.tsx

```diff
@@ -0,0 +1,11 @@
+import { CreateAccount } from "./createAccount";
+import { Login } from "./login";
+
```

Comment:
With #139 , we are trying to ensure that we include SEO metadata for each page. Are you able to add something similar to [this block](https://github.com/limlabs/saleor-storefront-starter/blob/main/src/app/%5Blocale%5D/%5Bchannel%5D/c/all/page.tsx#L28), but with translations for this page?